src/common/res/features/budget-progress-bars/pacing.js

```diff
@@ -0,0 +1,74 @@
+(function poll() { 
+  // Waits until an external function gives us the all clear that we can run (at /shared/main.js)
+  if ( typeof ynabToolKit !== "undefined"  && ynabToolKit.pageReady === true ) {
+  
+    ynabToolKit.featureOptions.budgetProgressBars = true;
+    ynabToolKit.budgetProgressBars = function ()  { // Keep feature functions contained within this
+    	var entityManager = ynab.YNABSharedLib.defaultInstance.entityManager;
+
+	   	function getCalculation(subCategoryName) {
+    		var crazyInternalId = "mcbc/" + ynabToolKit.parseSelectedMonth().yyyymm() + "/" + entityManager.getSubCategoryByName(subCategoryName).getEntityId();
+			var calculation = entityManager.getMonthlySubCategoryBudgetCalculationById(crazyInternalId);
```

Comment:
After sticking in some console.log statements in my build it appears that this is returning a default object, at least everything is 0 in the object returned. This explains why progress bars were not visualizing for me.